tests/test_acceptance.py

```diff
@@ -258,16 +294,37 @@ class Meta:
         self.assertRaises(ValidationError, s2.validate)
 
     def test_missing_fields(self):
-        class Obj(object):
+        class DefaultModelObj(object):
             def __init__(self):
                 self.string_field = "strfield"
 
         class ObjSerializer(Serializer):
             class Meta:
-                model = Obj
+                model = DefaultModelObj
 
             string_field = StringField(name="stringField", required=False, allow_null=False)
 
         s = ObjSerializer(data={})
         s.validate()
-        self.assertEqual(s.object.string_field, Obj().string_field)
+        self.assertEqual(s.object.string_field, DefaultModelObj().string_field)
```

Comment:
I think I'd rather leave this test intact and write a new one to test the new case you are trying to cover here.